src/HelloPublisher/Program.cs

```diff
@@ -1,17 +1,19 @@
-﻿using Azure;
+using Azure;
```

Comment:
Wait this file is not meant to be part of the source code right?

Reply:
I didn't really tought too much about the code. I focused on the pipeline. Should we create a task to revisit the code?

Reply:
Yeah we could have a story to just prettify what's needed